### PR TITLE
Add support for CAGetConfiguration (399)

### PR DESCRIPTION
**Pull request by larrytamnjong**

### CODE REVIEW

File: Consul.Test/ConnectTest.cs [hunk 1]
```diff
@@ -41,5 +41,14 @@ public async Task Connect_CARoots()
...
             Assert.NotNull(root.SigningKeyID);
         }
+
+        [Fact]
+        public async Task Connent_GetCAConfigurationTest()
+        {
+            var req = await _client.Connect.CAGetConfig();
+            var result = req.Response;
+
+            Assert.Equal("consul", result.Provider);
```
Comment (marcin-krystianc) [inline Consul.Test/ConnectTest.cs:51]: Please test all other fields too.

**Comment by larrytamnjong** [comment]
Hi @marcin-krystianc please can you have another look and let me know if this is okay, thanks :)

File: Consul.Test/ConnectTest.cs [hunk 2]
```diff
@@ -41,5 +41,18 @@ public async Task Connect_CARoots()
             Assert.NotNull(root.RootCert);
             Assert.NotNull(root.SigningKeyID);
         }
+
+        [Fact]
+        public async Task Connent_GetCAConfigurationTest()
```
Comment (marcin-krystianc) [inline Consul.Test/ConnectTest.cs:46]: A typo: 
```suggestion
        public async Task Connect_GetCAConfigurationTest()
```

**Review by marcin-krystianc — COMMENTED** [review]
There is a typo in the test name, looks good otherwise.

**Review by marcin-krystianc — APPROVED** [review]

**Pull request merged**